### DISCUSSION TITLE
Archive File

### DISCUSSION THREAD

**Discussion by Azurewren** [therden/packrat#5] [Q&A] [answered]
Can the filepath to the archive file use templated variables such as Obsidian's {{date}} or Templater's <%tp.date.now("YYYY-MM-DD")%>?

**Comment by therden** [comment, marked as answer]
Hello @Azurewren.  

Thank you for your interest in this plugin.

Template variables in the Archive filepath aren't supported at this time, but I'll create an Issue from this discussion with supporting Obsidian's built-in {{date}} as a feature request.  I will need to look into how involved that might be to implement; I only code as a personal hobby, and this plugin was my first foray into writing javascript/typescript.  

(I am not inclined to support the _Templater_ variable, as I don't see what it would add in this use-case if {{date}} support is added.  But if you feel otherwise, please feel free to follow-up!)

**Reply by Azurewren** [reply]
Hmm.. basically I was looking for a way to have multiple archive files. Perhaps there could be a way to define what archive file to place a task in, otherwise use the default archive file defined in the settings?
